scrabble.js

```diff
@@ -1,8 +1,81 @@
 var Scrabble = function() {};
 
-// YOUR CODE HERE
-Scrabble.prototype.helloWorld = function() {
-  return 'hello world!';
+Scrabble.prototype.letters = {
+  "a": 1, "b": 3, "c": 3, "d": 2, "e": 1, "f": 4, "g": 2,
+    "h": 4, "i": 1, "j": 8, "k": 5, "l": 1, "m": 3, "n": 1, "o": 1, "p": 3,
+    "q": 10, "r": 1, "s": 1, "t": 1, "u": 1, "v": 4, "w": 4, "x": 8, "y": 4,
+    "z": 10
+};
+
+// Scrabble.prototype.helloWorld = function() {
+//   return 'hello world!';
+// };
+
+Scrabble.prototype.score = function(word) {
+  word = word.toLowerCase();
+  var score = 0;
+  if (word === "" || word === " ") {
```

Comment:
there must be a javascript function that does this for you, like ```.isEmpty```. I don't know exactly the function, since I haven't looked at javascript in months, but you could look that up